app/controllers/invoices_controller.rb

```diff
@@ -49,7 +50,17 @@ def load_printable_procedure_field_values
       .joins(:field)
       .includes(:field)
       .where(
-        procedures_template_fields: { printable: true },
+        procedures_template_fields: { printable: true, print_position: 0 },
```

Comment:
usa las constantes de los enums que te puse como comentario en el modelo:

![Screen Shot 2023-07-21 at 11 05 45](https://github.com/Notarias/Notarias/assets/1482411/b1415161-1931-4f98-b92c-65cb697cd018)

```suggestion
        procedures_template_fields: { printable: true, print_position: Procedures::TemplateField::TOP_POSITION },
```

Reply:
resolved